### PR TITLE
#236: Update README for Android development

### PR DESCRIPTION
**Pull request by enieber**
create the step by step to run in android to clone issue #236

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
...
-Run Mobidex in an Android emulator:
+Start develop in Android:
 
-1.  In the terminal, execute `npm start`.
-2.  Open android studio and import mobidex from the `android` directory
-3.  Comment out `android.enableAapt2=false` in `gradle.properties`
-4.  Sync gradle files
-5.  Click play!
+ 0. The first steps is your workstation is installed dependencies based in [react-native documentation](https://facebook.github.io/react-native/docs/getting-started) or follow the [this steps](#setup).
+ 1. install dependencies of project using yarn with command `yarn` or npm `npm install`.
```
Comment (generalpiston) [inline README.md]: Yarn should not work.

File: README.md [hunk 2]
```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
...
 
-Run Mobidex in an Android emulator:
+Start develop in Android:
 
-1.  In the terminal, execute `npm start`.
-2.  Open android studio and import mobidex from the `android` directory
-3.  Comment out `android.enableAapt2=false` in `gradle.properties`
-4.  Sync gradle files
-5.  Click play!
+ 0. The first steps is your workstation is installed dependencies based in [react-native documentation](https://facebook.github.io/react-native/docs/getting-started) or follow the [this steps](#setup).
```
Comment (generalpiston) [inline README.md]: "Make sure your system is setup for react-native"

File: README.md [hunk 3]
```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
...
+ 1. install dependencies of project using yarn with command `yarn` or npm `npm install`.
+ 2. Connect the smartphone or start the your favorite emulator.
+ 3. Run the comand `react-native run-android` to compile and run the install project in android.
```
Comment (generalpiston) [inline README.md]: AFAIK this does not work

File: README.md [hunk 4]
```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
...
+ 4. Run the command `sudo rm -r node_modules/web3-contracts-loader/node_modules/web3`
```
Comment (generalpiston) [inline README.md]: Why is this needed?

File: README.md [hunk 5]
```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
...
+ 5. Run the command `sudo cp -R node_modules/web3/ node_modules/web3-contracts-loader/node_modules/`
```
Comment (generalpiston) [inline README.md]: Same

File: package.json [hunk 6]
```diff
@@ -56,13 +56,13 @@
...
         "react-native-randombytes": "^3.0.0",
-        "react-native-restart": "0.0.6",
+        "react-native-restart": "0.0.10",
         "react-native-router-flux": "^4.0.0-beta.28",
         "react-native-svg": "^9.4.0",
         "react-native-svg-charts": "^5.0.0",
         "react-native-swipeout": "^2.3.3",
         "react-native-uuid-generator": "^4.0.1",
-        "react-native-vector-icons": "^4.5.0",
+        "react-native-vector-icons": "^6.5.0",
```
Comment (generalpiston) [inline package.json]: Unnecessary?

File: package.json [hunk 7]
```diff
@@ -78,9 +78,11 @@
...
         "underscore": "^1.8.3",
         "url": "~0.10.1",
-        "web3": "^1.0.0-beta.36",
-        "web3-contracts-loader": "github:generalpiston/web3-contracts-loader",
-        "web3-provider-engine": "github:generalpiston/web3-provider-engine#patch-1"
+        "web-core": "^0.0.4",
+        "web3": "^1.0.0-beta.21",
+        "web3-contracts-loader": "https://github.com/generalpiston/web3-contracts-loader.git",
+        "web3-core": "^1.0.0-beta.50",
+        "web3-provider-engine": "web3-provider-engine@15.0.0"
```
Comment (generalpiston) [inline package.json]: Seems like all of the package changes are unnecessary?

File: README.md (hunk 1 above)
Comment (generalpiston) [reply README.md]: Let's yank Yarn and do it in a separate issue?

File: package.json [hunk 8]
```diff
@@ -78,8 +78,10 @@
         "shelljs": "^0.8.3",
         "underscore": "^1.8.3",
         "url": "~0.10.1",
-        "web3": "^1.0.0-beta.36",
-        "web3-contracts-loader": "github:generalpiston/web3-contracts-loader",
+        "web-core": "^0.0.4",
+        "web3": "^1.0.0-beta.21",
```
Comment (generalpiston) [inline package.json]: All of these dependencies shouldn't change.

File: README.md (hunk 4 above)
Comment (generalpiston) [reply README.md]: What error are you getting? That shouldn't matter.

Comment (enieber) [reply README.md]: I have much conflict with different versions of web3 in web-contracts-loader and like the dependency of Mobidex project

File: README.md [hunk 9]
```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
...
-Run Mobidex in an Android emulator:
+Start develop in Android:
 
-1.  In the terminal, execute `npm start`.
-2.  Open android studio and import mobidex from the `android` directory
-3.  Comment out `android.enableAapt2=false` in `gradle.properties`
-4.  Sync gradle files
-5.  Click play!
+ 0. Make sure your system is setup for react-native.
+ 1. install dependencies of project with command `npm install`.
```
Comment (generalpiston) [inline README.md]: Capitalize `i` in `install`

File: README.md [hunk 10]
```diff
@@ -148,15 +149,16 @@ Then, select your device simulator of choice and click the **play** icon at the
...
+ 2. Connect the smartphone or start the your favorite emulator.
+ 3. Run the comand `react-native run-android` to compile and run the install project in android.
+ 4. Run the command `sudo rm -r node_modules/web3-contracts-loader/node_modules/web3`
+ 5. Run the command `sudo cp -R node_modules/web3/ node_modules/web3-contracts-loader/node_modules/`
+ 6. Run the command `react-native start` to run server js and loading the MobiDex app.
```
Comment (generalpiston) [inline README.md]: `MobiDex` => `Mobidex`

File: README.md (hunk 4 above)
Comment (generalpiston) [reply README.md]: We can fix this in a separate issue. Please create one.

**Comment by enieber** [comment]
@generalpiston can be review?
